plugins/hls-fourmolu-plugin/src/Ide/Plugin/Fourmolu.hs

```diff
@@ -134,13 +135,12 @@ provider recorder plId ideState typ contents fp fo = withIndefiniteProgress titl
                                 { _type_ = MessageType_Error
                                 , _message = errorMessage
                                 }
-                        return . Left $ responseError errorMessage
+                        return . Left $ PluginInternalError errorMessage
```

Comment:
etc.

Reply:
This is also not inside an ExceptT block, so throwError won't work. (unless we convert it to use ExceptT that is...)